### ISSUE TITLE
reverse lat/long in the settings page

### ISSUE THREAD

**Issue by snoop168** [tananaev/traccar#1482]
I think the titles are incorrect on your fields regarding LAT/LONG on the Settings -> Account window. I put in our lat and long (then log out and back in) and it puts me in Antartica. If I reverse what our LAT/LONG should be it puts me in the right spot. I double checked using this site http://www.darrinward.com/lat-long/

Can someone else check and confirm they have the same problem?

**Comment by tananaev** [comment]
Yes, I can confirm that it's a known problem in version 3.1. It's already fixed in the code. Just need to wait for the next release.
